PipeableTests/PipeableElement_ClickTests.swift

```diff
@@ -37,6 +89,76 @@ final class PipeableElementClickTests: PipeableXCTestCase {
         }
     }
 
+    func testClickThenOnBlur() async throws {
+        let page = PipeablePage(webView)
+
+        _ = try? await page.goto("\(testServerURL)/click.html")
+
+        let onBlursBefore = try await page.evaluate("window.onBlurs")
+        guard let numOnBlursBefore = onBlursBefore as? Int else {
+            XCTFail("Could not read on blurs")
+            return
+        }
+
+        // type some text
+        let textInEl = try await page.querySelector("#text_input")
+        try await textInEl?.focus()
+        try await textInEl?.type("hello")
+
+        // click on another element to make sure onblur is fired for our text input
+        let linkEl = try await page.querySelector("#normal_link")
+        try await linkEl?.click()
+
+        let onBlursAfter = try await page.evaluate("window.onBlurs")
+        guard let numOnBlursAfter = onBlursAfter as? Int else {
+            XCTFail("Could not read on blurs")
+            return
+        }
+
+        XCTAssertEqual(numOnBlursAfter, numOnBlursBefore + 1, "Blur not registered")
+    }
+
+    func testClickOnContainerWithDisabledElementInside() async throws {
```

Comment:
@pdobrev this test mostly just records that we have a slight difference in behavior between us and Playwright, if you want I am happy to update the click code so that it matches the behavior of Playwright. Let me know what you think!

Reply:
I think our design goal should be to have the behavior match playwright / puppeteer as closely as we can. 

Thanks for thoroughly testing, we should address it, but that can also be part of another PR (since this one was more about focusing), but happy to ship it as part of this one.